Cargo.toml

```diff
@@ -22,6 +22,8 @@ futures = "0.3"
 
 itertools = ">=0.10.0"
 object_store = { version = "0.10.2", features = ["aws", "azure", "gcp"] }
+
+serde = "1.0.156"
```

Comment:
We should use this in `[dependencies]` below.

Reply:
@gruuya I get an error if I do not set it here, as I'm inheriting it in the `object_store_factory` Cargo.toml. Btw, it is also set in the `[dependecies]`.

The error is: 

```
error: failed to load manifest for workspace member `/Users/lizardo/Workspace/seafowl/object_store_factory`
referenced by workspace at `/Users/lizardo/Workspace/seafowl/Cargo.toml`

Caused by:
  failed to parse manifest at `/Users/lizardo/Workspace/seafowl/object_store_factory/Cargo.toml`

Caused by:
  error inheriting `serde` from workspace root manifest's `workspace.dependencies.serde`

Caused by:
  `dependency.serde` was not found in `workspace.dependencies
```

Reply:
No, i meant for serde to remain in `workspace.dependencies`, just to change line 129/131 in this file to inherit from it `serde = { workspace = true }`.

Reply:
Ahh, got it.